modules/core/test/ocl/test_arithm.cpp

```diff
@@ -867,7 +870,10 @@ OCL_TEST_P(AddWeighted, Mat)
         OCL_OFF(cv::addWeighted(src1_roi, alpha, src2_roi, beta, gama, dst1_roi));
         OCL_ON(cv::addWeighted(usrc1_roi, alpha, usrc2_roi, beta, gama, udst1_roi));
 
-        Near(3e-4);
+    if(dst1_roi.depth() >= CV_32F)
```

Comment:
Thanks for patch update! Looks good to me :+1:

Please fix `if`/`else` indentation here.

Reply:
Ah, right. Thank you for such a fine review !